capstone/src/main/webapp/cover.css

```diff
@@ -0,0 +1,89 @@
+/* Links */
+a, a:focus, a:hover {
+  color: #fff;
+}
+
+/* Custom default button */
+.btn-secondary, .btn-secondary:hover, .btn-secondary:focus {
+  color: #fff;
+  text-shadow: none; /* Prevent inheritance from `body` */
+  background-color: #451735;
+  border: .05rem solid #fff;
+}
+
+/* Base structure */
+html, body {
+  height: 100%;
+  background-image: url(/assets/Pattern-Randomized.svg);
+}
+
+body {
+  display: -ms-flexbox;
+  display: flex;
+  color: #451735;
+}
+
+.cover-container {
+  max-width: 80em;
+}
+
+/* Header */
+.masthead {
+  margin-bottom: 2rem;
+}
+
+.masthead-brand {
+  margin-bottom: 0;
+}
+
+.nav-masthead .nav-link {
```

Comment:
Should there be a comma between these two elements? I see there are commas in some places